tests/baselines/reference/jsdocTypeTagCast.errors.txt

```diff
@@ -69,25 +69,25 @@ tests/cases/conformance/jsdoc/b.js(67,8): error TS2454: Variable 'numOrStr' is u
     someBase = /** @type {SomeBase} */(someDerived);
     someBase = /** @type {SomeBase} */(someBase);
     someBase = /** @type {SomeBase} */(someOther); // Error
-                   ~~~~~~~~~~~~~~~~
+                   ~~~~~~~~~~~~~~~~~
 !!! error TS2352: Conversion of type 'SomeOther' to type 'SomeBase' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
 !!! error TS2352:   Property 'p' is missing in type 'SomeOther' but required in type 'SomeBase'.
 !!! related TS2728 tests/cases/conformance/jsdoc/b.js:17:9: 'p' is declared here.
     
     someDerived = /** @type {SomeDerived} */(someDerived);
     someDerived = /** @type {SomeDerived} */(someBase);
     someDerived = /** @type {SomeDerived} */(someOther); // Error
-                      ~~~~~~~~~~~~~~~~~~~
+                      ~~~~~~~~~~~~~~~~~~~~
 !!! error TS2352: Conversion of type 'SomeOther' to type 'SomeDerived' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
 !!! error TS2352:   Type 'SomeOther' is missing the following properties from type 'SomeDerived': x, p
     
     someOther = /** @type {SomeOther} */(someDerived); // Error
-                    ~~~~~~~~~~~~~~~~~
+                    ~~~~~~~~~~~~~~~~~~
 !!! error TS2352: Conversion of type 'SomeDerived' to type 'SomeOther' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
 !!! error TS2352:   Property 'q' is missing in type 'SomeDerived' but required in type 'SomeOther'.
 !!! related TS2728 tests/cases/conformance/jsdoc/b.js:28:9: 'q' is declared here.
     someOther = /** @type {SomeOther} */(someBase); // Error
-                    ~~~~~~~~~~~~~~~~~
+                    ~~~~~~~~~~~~~~~~~~

```

Comment:
This seems worse

Reply:
Yes, but the problem is that the error message uses the entire tag as the errorNode, when it should just use the type (something like typeTag.typeExpression.type). (Tags now correctly include their entire span, even the trailing white space, before the next tag/end of comment.)

There's some followup work to improve some error messages for sure.

Reply:
Later: this is now done.